### DISCUSSION TITLE
any plans for a update to html to come ?

### DISCUSSION THREAD

**Discussion by guno1928** [MyreMylar/pygame_gui#385] [General]
wanted to know if any more support for the html would come. having more tags and options would be great

**Comment by MyreMylar** [comment]
Hello,

Any specific tags or options you are after? I tend to add things as there is need/desire. The 'HTML subset' used for text formatting is HTML inspired (inspired by older versions of HTML actually) rather than matching any particular HTML spec.

**Reply by scribblecrumb** [reply]
If I may interject, text alignment (center, left, right, and justified) would be helpful, as well as perhaps  horizontal rule?

**Reply by guno1928** [reply]
> Hello,
> 
> Any specific tags or options you are after? I tend to add things as there is need/desire. The 'HTML subset' used for text formatting is HTML inspired (inspired by older versions of HTML actually) rather than matching any particular HTML spec.

maybe some meta tags ? and also some more functions into the current tags like margin, padding, justify-content etc. and maybe some audio support

**Reply by MyreMylar** [reply]
> If I may interject, text alignment (center, left, right, and justified) would be helpful, as well as perhaps horizontal rule?

horizontal rule `<hr>` is actually in already, I just need to update the documentation.

**Reply by guno1928** [reply]
@MyreMylar a addition to this topic would buttons be a thing of the future within the html ?
